src/main/java/md2k/mcerebrum/cstress/autosense/AUTOSENSE.java

```diff
@@ -49,6 +49,13 @@
     public static final int QUALITY_MISSING = 4;
     public static final int QUALITY_BAND_LOOSE = 2;
 
+    public static final int QUALITY_acceptableOutlierPercent = 34;
+    public static final int QUALITY_outlierThresholdHigh = 4000;
```

Comment:
Most of these should be based on a ratio (e.g. 0 <= x <= 1) and combined with the ADC range in the logic.  This ADC should be specified as an input to the processing module